main.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from PIL import Image
+from pathlib import Path
+from random import randint
+valid_images = [".jpg", ".jpeg", ".png"]
```

Comment:
blank line between imports and code. (general tip: ctrl+alt+l formats code in pycharm)
also, const names should be all upper case (e.g. VALID_IMAGES)

Reply:
done